Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -1642,6 +1642,13 @@ - (void)didPressSendButton:(UIButton *)button
     }
 }
 
+- (void)finishSendingMessageAnimated:(BOOL)animated
+{
+    [super finishSendingMessageAnimated:animated];
```

Comment:
Though this is more concise, I believe the reason the scroll logic was in yapDatabaseModified was because not until then does the new cell get inserted into the collectionView. 

How can we be sure the interaction has been inserted into the collectionView at this point?